api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -55,6 +64,47 @@ public void setSkipHardGoalCheck(boolean skipHardGoalCheck) {
         this.skipHardGoalCheck = skipHardGoalCheck;
     }
 
+    @Description("The upper bound of ongoing replica movements going into/out of each broker")
+    @Minimum(0)
+    public int getConcurrentPartitionMovementsPerBroker() {
+        return concurrentPartitionMovementsPerBroker;
+    }
+
+    public void setConcurrentPartitionMovementsPerBroker(int movements) {
+        this.concurrentPartitionMovementsPerBroker = movements;
+    }
+
+    @Description("The upper bound of ongoing replica movements between disks within each broker")
+    @Minimum(0)
+    public int getConcurrentIntraBrokerPartitionMovements() {
+        return concurrentIntraBrokerPartitionMovements;
+    }
+
+    public void setConcurrentIntraBrokerPartitionMovements(int movements) {
+        this.concurrentIntraBrokerPartitionMovements = movements;
+    }
+
+    @Description("The upper bound of ongoing leadership movements")
```

Comment:
what's kind of "leadership" this is referring? "leader" partition movements? can you make it clear?